src/systems/ec/implementations/vanilla/packages/ec-component-blockquote/ec-component-blockquote.scss

```diff
@@ -24,7 +24,35 @@
   }
 }
 
+@mixin ecl-blockquote-print() {
+  @media print {
+    .ecl-blockquote__body {
+      color: #000;
+      font: $ecl-font-prolonged-print-m;
+    }
+
+    .ecl-blockquote__author {
+      color: #000;
+      font: $ecl-font-prolonged-print-m;
+      font-weight: $ecl-font-weight-print-bold;
+    }
+
+    /* stylelint-disable-next-line order/order */
+    @include ecl-media-breakpoint-up('md') {
```

Comment:
breakpoints in print?

Reply:
btw, the print styles should have been in `ec-component-blockquote-print.scss` :)

Reply:
This component was just a first test, I forgot to remove these rules... Please ignore them